shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Value.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-unused-binds #-}
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
+
+module Cardano.Ledger.Value where
```

Comment:
This is in the wrong place, and is in any case already around:

https://github.com/input-output-hk/cardano-ledger-specs/blob/master/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Value.hs

Changes should be made there.

Reply:
This is a work in progress, when I have made everyone happy, before we merge with master I will do that.